src/components/help/help.test.js

```diff
@@ -219,4 +219,128 @@ context("Tests for Help component", () => {
       }
     );
   });
+
+  describe("Accessibility tests for Help component", () => {
+    it.each(testData)("should pass accessibilty tests for helpId", (id) => {
```

Comment:
Why is the terminology different for this test case compared to the others? It would be better for this to read "should check accessibility for helpId".